app/presenters/idv/in_person/verified_presenter.rb

```diff
@@ -0,0 +1,22 @@
+module Idv
+  module InPerson
+    class VerifiedPresenter
+      # update to user's time zone when out of pilot
+      USPS_SERVER_TIMEZONE = ActiveSupport::TimeZone['America/New_York']
```

Comment:
It's a problem we can sort out later, but I'm wondering how we'll know what the user's timezone is. I don't think we record this preference anywhere.